doc/operator-application-capabilities.md

```diff
@@ -1278,6 +1278,34 @@ spec:
     name: tenant-secret
 ```
 
+You can set the following optional parameters in your {ProductShortName} Tenant via the `Tenant` CR:
```

Comment:
"{ProductShortName}" ?

Reply:
oh, sorry, copied ...
Updated. Thank you